Source/DafnyCore/Compilers/Compiler-go.cs

```diff
@@ -3491,7 +3505,7 @@ private class SpecialNativeType : UserDefinedType {
       } else {
         Contract.Assert(ct is SeqType);  // follows from precondition
         if (ct.Arg.IsCharType) {
-          wr.Write("_dafny.SeqOfChars");
+          wr.Write(UnicodeChars ? "_dafny.SeqOfCodePoints" : "_dafny.SeqOfChars");
```

Comment:
Also, with literals, shouldn't we be able to write `[]rune(<literal>)`?

Reply:
We do something like that for literals, yes, but this is a collection display, so at best something like `['H', 'e', 'l', 'l', 'o']`. I don't think it's worth special-casing a sequence of characters into a string literal.